### PR TITLE
GT Serviços: feat(Payments): PB42  - PB55  - PB56  - PB57  - PB58  - PB59  - PB60  - PB61  - SB28 

### PR DESCRIPTION
**Pull request by andretrindade**
**Propostas contempladas nesse pull request**

- PB42 
- PB55 
- PB56 
- PB57 
- PB58 
- PB59 
- PB60 
- PB61 
- SB28

### CODE REVIEW

**Review by CeciliaFFernandes — APPROVED** [review]

File: swagger-apis/payments/2.1.0.yml [hunk 1]
```diff
@@ -0,0 +1,2596 @@
...
+              maxLength: 10
+              pattern: '^(\d{4})-(1[0-2]|0?[1-9])-(3[01]|[12][0-9]|0?[1-9])$'
+              example: '2021-01-01'
+              description: |
+                Define a data alvo da liquidação do pagamento. 
+
+                O fuso horário de Brasília deve ser utilizado para criação e racionalização sobre os dados deste campo.
+
+                Observação: Esse campo deverá sempre ser no mínimo D+1 corrido, ou seja, a data imediatamente posterior em relação a data do consentimento considerando o fuso horário de Brasília e deverá ser no máximo D+365 corridos a partir da data do consentimento considerando o fuso horário de Brasília
+    XFapiInteractionId:
```
Comment (guilhermedecampo) [inline swagger-apis/payments/2.1.0.yml:2229]: This should be required?

File: swagger-apis/payments/2.1.0.yml [hunk 2]
```diff
@@ -0,0 +1,2596 @@
...
+        '529':
+          $ref: '#/components/responses/SiteIsOverloaded'
+        default:
+          description: Erro inesperado.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseError'
+      security:
+        - OAuth2AuthorizationCode:
```
Comment (guilhermedecampo) [inline swagger-apis/payments/2.1.0.yml:340]: This should be client credentials, no?

File: swagger-apis/payments/2.1.0.yml (hunk 1 above)
Comment (andretrindade) [reply swagger-apis/payments/2.1.0.yml:2229]: We have two XFapiInteractionId.

1. It's a component into structure parameter/XFapiInteractionId which responsable just for parameter of endpoints . **It componet is required.**
2. It's a component into structure scheme/XFapiInteractionId which shared with others componentes like InternalServerError

Comment (guilhermedecampo) [reply swagger-apis/payments/2.1.0.yml:2229]: Gotcha, so all good?

**Pull request merged**